internetarchive/openlibrary
# Add string format validation to `i18n-messages validate`

Adds built-in string format validation to `i18n-messages validate`

### Technical

Restructures code by creating a directory of validator objects.  Each validator requires a Babel message and catalog in instantiation, and has a `validate` method that checks a single message and returns a list of error strings if message is invalid.

### Testing

1. Execute `i18n-messages validate` using a locale with known format string errors.
2. Ensure that error messages displayed identify all messages with errors.

### Screenshot


### Stakeholders

@cdrini
